lib/SARIF/Log.h

```diff
@@ -0,0 +1,36 @@
+//
+// Created by Daniel Dominguez on 11/1/23.
+//
+
+#ifndef VANGUARD_LOG_H
+#define VANGUARD_LOG_H
+
+#include "Object.h"
+#include "Run.h"
+#include <vector>
+#include <memory>
+
+constexpr auto SARIF_VERSION = "2.1.0";
+
+namespace vanguard {
+    class SARIF::Log : public SARIF::Object {
+    public:
+        using Run = SARIF::Run;
+        void addRun(std::unique_ptr<Run> run) {
+            runs.push_back(std::move(run));
+        }
+
+        std::string getVersion() {
+            return SARIF_VERSION;
+        }
+
+        std::vector<std::unique_ptr<Run>> getRuns() {
```

Comment:
nit: rename to indicate that the variable is being moved.
```suggestion
        std::vector<std::unique_ptr<Run>> takeRuns() {
```